### ISSUE TITLE
Feature: S3 compatible api endpoint for Atomic Drive

### ISSUE THREAD

**Issue by AlexMikhalev** [atomicdata-dev/atomic-data-rust#542]
Give the user the ability to generate S3-compatible credentials: 
AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, and 
the ability create AWS compatible AWS_ENDPOINT to the user can populate Atomic Drive using standard AWS S3 compatible tools like rclone (or numerous UI clients).

**Comment by joepio** [comment]
Does this mean letting usser access their atomic data as if atomic-server was an S3 bucket? Or something different?

And how does this relate to #543?

**Comment by AlexMikhalev** [comment]
Yes, like if you want to put 100 images into atomic server you can use rclone (or cloud drive), or `aws s3 cp` command.  It's not related to #543 - in 543 atomic server acts as a client for S3 compatible store, in this scenario Atomic Server acts like a s3 server i.e. minio/seaweedfs.

That may not only allow users to upload their assets quickly, but presents an interesting backup/sync mechanism: if you want to backup your data from one atomic server to another, you can use existing S3 tooling  - plenty of tools.
